dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/subcommands/impl/ParentTest.java

```diff
@@ -109,6 +109,55 @@ public void testForkRepositoriesFound() throws Exception {
         assertEquals(repoMap.size(), 3);
     }
 
+    @Test
+    public void forkRepositoriesFoundAndGetPathToDockerfiles_unableToforkRepo() throws Exception {
+        /**
+         * Suppose we have multiple dockerfiles that need to updated in a repo and we fail to fork such repo,
+         * we should not add those repos to pathToDockerfilesInParentRepo.
+         */
+        DockerfileGitHubUtil dockerfileGitHubUtil = mock(DockerfileGitHubUtil.class);
+
+        GHRepository contentRepo1 = mock(GHRepository.class);
+        when(contentRepo1.getFullName()).thenReturn("1");
+
+        GHRepository contentRepo2 = mock(GHRepository.class);
```

Comment:
Better name for this? `duplicateContentRepo1` maybe?

Reply:
Sounds good. I will update.